### DISCUSSION TITLE
Searching for part of a university name

### DISCUSSION THREAD

**Discussion by boothym** [osm-search/Nominatim#2272] [Geocoding around the world]
Came across this note: https://www.openstreetmap.org/note/2384794

https://nominatim.openstreetmap.org/ui/search.html?q=University+of+the+West+of+Scotland

Like the person who opened the note, I would have expected Nominatim to find the university (and others containing the search term): https://www.openstreetmap.org/way/4543895

But it seems like Nomination is looking for an exact match of the search term, rather than finding universities in OSM which begin with or contain the search term? 

I also searched for other universities and colleges which had "- XYZ Campus" in their name e.g. "ABC College - XYZ campus" and got similar results like a bus stop named "ABC College" but not the campus.

Is there anything that can or should be done with the OSM tagging? Or is this a Nominatim issue?

**Comment by mtmail** [comment]
Specifically for universities in Germany they're sometimes tagged as relation to collect the various campuses/faculties/buildings across a town, examples

https://www.openstreetmap.org/relation/8727202
https://www.openstreetmap.org/relation/4745499
https://www.openstreetmap.org/relation/1590189

**Reply by boothym** [reply]
Well that's seems to be the way to do it in a city, but in this case the campuses are a fair distance away. Here's an example: https://overpass-turbo.eu/s/169e - should these be connected in a relation and would it help Nominatim?

I suppose the issue being highlighted is why does Nominatim not return a partial match for the university name with "- Campus" on the end?

**Comment by lonvia** [comment]
The [wiki for amenity=university](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Duniversity) states very clearly that the tag is for the whole university not for individual campuses and I tend to agree. As a general rule, if the name contains multiple parts (here: university name + campus name) then there is a high probability that the tagging is off.

The discussion how to tag a campus vs. the whole of the university belongs to the tagging mailinglist, though.

**Reply by boothym** [reply]
Perhaps this is more of an issue about how Nominatim handles names with multiple parts?

For example, this town has two parts to its name but if I search for the either the first or second parts it doesn't find them: https://www.openstreetmap.org/search?query=elie#map=15/56.1905/-2.8202

**Reply by lonvia** [reply]
With this kind of tagging there is only so much that Nominatim can do about it. See discussions about abuse of the name tag for [hiking routes](https://github.com/osm-search/Nominatim/issues/413#issuecomment-207590647), [power lines](https://github.com/osm-search/Nominatim/issues/413#issuecomment-246389484) and [railway lines](https://github.com/osm-search/Nominatim/issues/391#issuecomment-214035172). This kind of stuff shouldn't need fixing on the data consumer side. It doesn't scale in the long term.
